Demo/EarlGreyExample/Podfile

```diff
@@ -19,5 +19,5 @@ end
 
 target 'EarlGreyExampleUITests' do
   #inherit! :search_paths
-  pod 'EarlGreyTest', :git => 'https://github.com/brettfazio/EarlGrey.git', :branch => 'recentclone'
+  pod 'EarlGreyTest', :git => 'https://github.com/google/EarlGrey.git', :branch => 'earlgrey2pd'
```

Comment:
? This is already pushed?

Reply:
Just to "earlgrey2pd" for testing purposes, it needs to exist somewhere for people to test it. 

We are not going to merge "earlgrey2pd"